jadx-core/src/main/java/jadx/core/plugins/JadxPluginManager.java

```diff
@@ -55,13 +55,13 @@ public void load() {
 	public void register(JadxPlugin plugin) {
 		Objects.requireNonNull(plugin);
 		PluginContext addedPlugin = addPlugin(plugin);
-		LOG.debug("Register plugin: {}", addedPlugin.getPluginId());
+		LOG.warn("Register plugin: {}", addedPlugin.getPluginId());
```

Comment:
I don't think these changes to log level are needed

Reply:
Ah true, it was some leftover stuff from me trying to get the plugin to load, will revert and check for other logs